README.md

```diff
@@ -47,16 +47,16 @@ As long as you know how to use numpy, pandas and so forth, you would probably kn
 
 All Xorbits APIs implemented or planned include:
 
-| API                                                                            | Implemented version or plan |
-|--------------------------------------------------------------------------------|-----------------------------|
-| [xorbits.pandas](https://doc.xorbits.io/en/latest/reference/pandas/index.html) | v0.1.0                      |
-| [xorbits.numpy](https://doc.xorbits.io/en/latest/reference/numpy/index.html)   | v0.1.0                      |
-| xorbits.sklearn                                                                | Planned in the near future  |
-| xorbits.xgboost                                                                | Planned in the near future  |
-| xorbits.lightgbm                                                               | Planned in the near future  |
-| xorbits.xarray                                                                 | Planned in the future       |
-| xorbits.scipy                                                                  | Planned in the future       |
-| xorbits.statsmodels                                                            | Planned in the future       |
+| API                                                                             | Implemented version or plan |
+|---------------------------------------------------------------------------------|-----------------------------|
+| [xorbits.pandas](https://doc.xorbits.io/en/latest/reference/pandas/index.html)  | v0.1.0                      |
+| [xorbits.numpy](https://doc.xorbits.io/en/latest/reference/numpy/index.html)    | v0.1.0                      |
+| xorbits.sklearn                                                                 | Planned in the near future  |
```

Comment:
Since xgb & lgm introduced earlier than sklearn, we can move this line below the lightgbm.

Reply:
done